htdocs/accountancy/admin/productaccount.php

```diff
@@ -465,15 +465,15 @@ function init_savebutton()
         	{
 	            console.log("We check if at least one line is checked")
 
-    			atleastoneselected=0;
+    			at leastoneselected=0;
```

Comment:
@ka7 are you sure you want to change here?

Reply:
@ka7 
"atleastoneselected"  this is a variable !!!